crates/typst-library/src/text/raw.rs

```diff
@@ -247,7 +262,34 @@ impl Synthesize for RawElem {
 impl Show for RawElem {
     #[tracing::instrument(name = "RawElem::show", skip_all)]
     fn show(&self, _: &mut Vt, styles: StyleChain) -> SourceResult<Content> {
-        let text = self.text();
+        let mut text = self.text();
+        if text.contains('\t') {
+            let tab_size = RawElem::tab_size_in(styles);
+            let amount = text.chars().filter(|&c| c == '\t').count();
```

Comment:
This would be more efficient to do directly on bytes, i.e. `text.bytes().filter(|&b| b == b'\t').count()`. But now it's probably me unnecessarily overoptimizing. :)

Reply:
I prefer this for readability. With bytes utf-8 knowledge would be required to verify.

Reply:
That's fair enough.